pravega/pravega-operator
# Issue 163: Create Helm chart to install Pravega

### Change log description

- [x] Update existing `pravega-operator` Helm chart
- [x] Add documentation for `pravega-operator` chart
- [x] Add tests for `pravega-operator` chart 
- [x] Create a `pravega` Helm chart to install a Pravega cluster with initial set of configuration parameters
- [x] Add documentation for `pravega` chart
- [x] Add tests for `pravega` charts

### Purpose of the change
- Fix #163 

### How to test the change
- Follow the instructions in the charts directory to install and check that the charts works as expected
- Tests should pass

---
Signed-off-by: Adrián Moreno <adrian@morenomartinez.com>
